### ISSUE TITLE
want "Fix DPI" in "Margins" panel

### ISSUE THREAD

**Issue by reznoa** [ScanTailor-Advanced/scantailor-advanced#93]
"Fix DPI" dialog is inconvenient when i need to fix many pages. (ex: scaned by mobile app)
It would be great if i could change the DPI in the "Margins" panel.

**Comment by pablogventura** [comment]
**“Fix DPI” exposed in the Margins panel** is still a **feature request** — not a committed roadmap item from this thread alone.

If you still want it, note whether DPI should follow **Output**, **Select Content**, or **global project** semantics; that affects where settings live.

Open for tracking.

**Comment by pablogventura** [comment]
Triage from current code:

The **Margins** (page layout) options widget already has a **Fix DPI** button (`fixDpiBtn`) that emits `fixDpiRequested` from `src/core/filters/page_layout/OptionsWidget.cpp`. That signal is wired in `src/app/MainWindow.cpp` to the same flow as the menu action (`fixDpiDialogRequested`), so DPI correction is reachable from the Margins panel today.

Question for the original request: is opening the existing **Fix DPI** dialog from Margins enough, or do you specifically need **inline DPI fields** (editable X/Y DPI directly in the Margins panel without a separate dialog)?

Depending on your answer, we can either close this as addressed by the button or scope a small UI follow-up PR.

**Comment by pablogventura** [comment]
There is **no open PR** for this issue yet: we are waiting on your answer to the earlier triage question (Fix DPI dialog from Margins vs inline DPI fields). Once the desired UX is confirmed, we can open a focused PR if still needed.

**Comment by reznoa** [comment]
i want inline DPI fields, if its okay.

It would be perfect if the focus were maintained on the DPI field even when moving to another page.

**Comment by pablogventura** [comment]
Thanks [@reznoa](https://github.com/reznoa) — that answers the triage question clearly.

We will treat this as accepted scope for a **follow-up PR**:

- **Inline DPI fields** in the Margins (page layout) panel (instead of relying only on the separate Fix DPI dialog for that workflow).
- **Keep keyboard focus on the DPI field** when switching pages, where technically feasible with the current page-switch / options-widget wiring.

I will link the PR here once it is opened. No need to repeat the request in a new issue.
